src/extension/flutter/flutter_daemon.ts

```diff
@@ -72,6 +73,18 @@ export class FlutterDaemon extends StdIOService<UnknownNotification> implements
 		}
 	}
 
+	public startPing() {
+		const intervalId = setInterval(async () =>  {
+			try {
+				const result = await withTimeout(this.daemonVersion(), "The daemon connection was lost", 1);
+			} catch (e) {
+				clearInterval(intervalId);
```

Comment:
We should probably ensure this is done on `dispose` too so when we unload we don't get an additional call (and a prompt) before it stops trying to ping. We could pull the `intervalId` up to the class, or do something like this:

```
this.disposables.push({ dispose: () => clearInterval(intervalId) });
```

Reply:
I added `intervalId` as a class var